aws/components/apigateway/id.ftl

```diff
@@ -1,7 +1,18 @@
 [#ftl]
 [@addResourceGroupInformation
     type=APIGATEWAY_COMPONENT_TYPE
-    attributes=[]
+    attributes=[
+        {
+            "Names" : "AccessLogging",
+            "Children" : [
```

Comment:
See below re adding a couple of extra children to switch between firehose and log groups.

Reply:
@ml019 I have incorporated your changes. Could you take another look?